functests/utils/mcps/mcps.go

```diff
@@ -144,7 +196,10 @@ func WaitForCondition(mcpName string, conditionType machineconfigv1.MachineConfi
 func WaitForProfilePickedUp(mcpName string, profileName string) {
 	EventuallyWithOffset(1, func() bool {
 		mcp, err := GetByName(mcpName)
-		Expect(err).ToNot(HaveOccurred(), "Failed getting MCP by name")
+		// we ignore the error and just retry in case of single node cluster
+		if err != nil {
```

Comment:
If I log, it will dump every 30 seconds, not sure it makes sense.
I'll put a log before and one after, I think it would serve the same purpose but it will be less intrusive